x-pack/plugins/cloud/server/util/generate_jwt.ts

```diff
@@ -5,55 +5,21 @@
  * 2.0.
  */
 
-import crypto from 'node:crypto';
-
-const toBase64 = (object: Record<string, unknown>): string => {
-  const str = JSON.stringify(object);
-  return Buffer.from(str).toString('base64');
-};
-
-const replaceSpecialChars = (b64string: string): string => {
-  // create a regex to match any of the characters =,+ or / and replace them with their // substitutes
-  return b64string.replace(/[=+/]/g, (charToBeReplaced) => {
-    switch (charToBeReplaced) {
-      case '=':
-        return '';
-      case '+':
-        return '-';
-      case '/':
-        return '_';
-      default:
-        return charToBeReplaced;
-    }
-  });
-};
-
-const generateJwtB64 = (object: Record<string, unknown>): string => {
-  const b64 = toBase64(object);
-  return replaceSpecialChars(b64);
-};
-
-const createSignature = (header: string, payload: string, secret: string): string => {
-  const signature = crypto.createHmac('sha256', secret);
-  signature.update([header, payload].join('.'));
-  const signatureStr = signature.digest('base64');
-  return replaceSpecialChars(signatureStr);
-};
+// eslint-disable-next-line import/no-extraneous-dependencies
+import jwt from 'jsonwebtoken';
```

Comment:
We should probably just add this dependency back to the `package.json` at this point, since alerting seems to be depending on it already anyway. 

cc @jportner I pinged you about this separately, would appreciate your eyes on it since security team had originally tried to remove this dep.

Reply:
@lukeelmers https://github.com/elastic/kibana/issues/110579

Reply:
I added the dependency to package.json and removed the eslint warnings.  @jportner let me know if I need to undo that, and revert to the manual process @SergeyPoluektov originally added.

Reply:
Thanks, we removed it simply because we were trying to get a handle on which dependencies were actually used in production / what is owned by our team, and we deemed it unnecessary at the time: #113723

[There aren't any security concerns with this dependency](https://snyk.io/test/npm/jsonwebtoken).

> We should probably just add this dependency back to the `package.json` at this point, since alerting seems to be depending on it already anyway.

Yep:

https://github.com/elastic/kibana/blob/113688274c5377d82c989a5db054b8739bc36fe8/x-pack/plugins/actions/server/builtin_action_types/lib/create_jwt_assertion.ts#L7-L8

We shouldn't ever be circumventing the `import/no-extraneous-dependencies` lint rule!